build.gradle

```diff
@@ -77,7 +79,7 @@ subprojects {
         kotlinOptions {
             languageVersion = "1.2"
             apiVersion = "1.2"
-            jvmTarget = "1.8"
+            jvmTarget = VERSION_1_8
```

Comment:
This sets the target for all JARs in the repo. Potentially we might target later versions for non-contract JARs, when the JAva 11 work is done?

Reply:
This _particular_ line just replaces "1.8" with its equivalent `JavaVersion` object, and "yes" this is configuring the Kotlin compiler to generate Java 8 byte-code across all sub-projects.

You would want to change this to `VERSION_11`, but have `deterministic.gradle` override it with `VERSION_1_8` for the `contract` and `money` modules.

Reply:
ok sounds good - just checking. Cheers!